PROJECT_PROGRESSION.md

```diff
@@ -96,14 +97,14 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
 - [ ] Add or Update Governance.md document (required for Impact stage)
 - [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
-- [ ] Project Charter is published on website or github
+- [ ] Publish Project Charter on website or github
 - [ ] Update legal copyright notice on project website and github
 - [ ] Add OpenJS Foundation logo to project website
 - [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
 - [ ] Transfer logomark to the OpenJS Foundation
 - [ ] If project is using crowdfunding platforms, add disclaimer to platforms
 - [ ] Identify individuals from the project to join the CPC
-- [ ] Document project and foundation contacts for:
+- [ ] The champion remains the main point of contact with the CPC, but also document project and foundation contacts for:
```

Comment:
```suggestion
- [ ] Document project and foundation contacts for:
```

Reply:
So rollback the change to the onboarding checklist?

Reply:
(Yes, sorry for confirming this with so much delay.)